indico/modules/events/models/persons.py

```diff
@@ -147,6 +147,12 @@ def principal(self):
             return EmailPrincipal(self.email)
         return None
 
+    @property
+    def is_principal_pending(self):
+        if self.principal is None:
```

Comment:
that check is not needed, `None` is perfectly fine in `isinstance` and will already return False in your case